### DISCUSSION TITLE
Could you add an indicator of how much space each docset requires on disk ?

### DISCUSSION THREAD

**Discussion by crillion** [zealdocs/zeal#1359] [Ideas]
Hi,

   I'd like to suggest you to add a feature for letting know, both before installation and also after, how much space a docset requires on disk, so that one can decide based on his disk space if he wants to install a particular topic.

 Thanks,

              Marco

**Comment by trollixx** [comment]
Tracked in https://github.com/zealdocs/zeal/issues/353.
